client/packages/system/src/Immunisation/ProgramView/ProgramView.tsx

```diff
@@ -1,29 +1,78 @@
-import React, { FC, useEffect } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import {
   TableProvider,
   createTableStore,
   useBreadcrumbs,
   NothingHere,
   createQueryParamsStore,
+  DataTable,
+  useNavigate,
+  useTranslation,
+  useUrlQueryParams,
+  useColumns,
 } from '@openmsupply-client/common';
 import { Toolbar } from './Toolbar';
+import { AppBarButtons } from './AppBarButtons';
 
 // dummy data
 const data = {
-  name: 'data',
+  name: 'some program name',
 };
 
+export interface Immunisation {
+  id: string;
+  name: string;
+  vaccines: string[];
+  isNew: boolean;
+}
```

Comment:
```suggestion
export interface VaccineCourse {
  id: string;
  name: string;
  targetDemographicName: string
  doses: number
}
```

Reply:
I think you're intending to use this type to disply the vaccine courses in a list view right?

Reply:
Yes correct!